src/main/resources/portal.properties.EXAMPLE

```diff
@@ -106,8 +106,11 @@ ldap.attributes.last_name=sn
 ldap.attributes.given_name=givenName
 ldap.attributes.email=mail
 
+# study view settings
 # always show studies with this group
 always_show_study_group=
+mdacc.heatmap.study.meta.url=http://bioinformatics.mdanderson.org/study2url?studyid=
```

Comment:
I suggest not doing anything this specific.  Then we won't have to update code just because we changed our server to https.  The tab is already hidden if the calls fails for any reason, including this condition.  Perhaps we should change the properties to not say "http:" and instead just use "//<server>". ?  I can test this.